src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -105,10 +112,10 @@
 
     // default value of 40MB based on a test with CountReads (it's 5x faster than no prefetching)
     @Argument(fullName = StandardArgumentDefinitions.CLOUD_PREFETCH_BUFFER_LONG_NAME, shortName = StandardArgumentDefinitions.CLOUD_PREFETCH_BUFFER_SHORT_NAME, doc = "Size of the cloud-only prefetch buffer (in MB; 0 to disable).", optional=true)
-    public int cloudPrefetchBuffer = getDefaultCloudPrefetchBufferSize();
+    public int cloudPrefetchBuffer = ConfigUtils.getFromConfig("cloudPrefetchBuffer");
 
     @Argument(fullName = StandardArgumentDefinitions.CLOUD_INDEX_PREFETCH_BUFFER_LONG_NAME, shortName = StandardArgumentDefinitions.CLOUD_INDEX_PREFETCH_BUFFER_SHORT_NAME, doc = "Size of the cloud-only prefetch buffer (in MB; 0 to disable). Defaults to cloudPrefetchBuffer if unset.", optional=true)
-    public int cloudIndexPrefetchBuffer = getDefaultCloudIndexPrefetchBufferSize();
+    public int cloudIndexPrefetchBuffer = ConfigUtils.getFromConfig("cloudPrefetchBuffer");
```

Comment:
I'm confused by these, here you use ConfigUtils.getFromConfig(), but below you declare a new getCloudPrefetchBufferSize() method which seems like it's a replacement for getDefaultCloudPrefetchBufferSize().